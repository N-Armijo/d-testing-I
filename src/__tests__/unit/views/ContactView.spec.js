import { describe, it, expect } from 'vitest'
import { mount } from '@vue/test-utils'
import AboutView from '@/views/ContactView.vue'

describe('ContactView.vue', () => {
  it('Contrasta Snapshot del HTML', () => {
    const wrapper = mount(AboutView)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
