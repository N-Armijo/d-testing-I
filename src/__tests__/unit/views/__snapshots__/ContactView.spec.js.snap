// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ContactView.vue > Contrasta Snapshot del HTML 1`] = `
"<div class="about">
  <h1>Contacto</h1>
</div>"
`;
